FrontDesk/src/FrontDesk.SharedKernel/Interfaces/IDomainEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace FrontDesk.SharedKernel.Interfaces
```

Comment:
Is this used, or do we just use BaseDomainEvent?

Reply:
Yes that not used.